packages/react-native-bridge/common/supported-blocks.json

```diff
@@ -0,0 +1,36 @@
+{
+	"common": [
+		"core/paragraph",
+		"core/heading",
+		"core/more",
+		"core/image",
+		"core/video",
+		"core/nextpage",
+		"core/separator",
+		"core/list",
+		"core/quote",
+		"core/media-text",
+		"core/preformatted",
+		"core/gallery",
+		"core/columns",
+		"core/column",
+		"core/group",
+		"core/freeform",
+		"core/button",
+		"core/spacer",
+		"core/shortcode",
+		"core/buttons",
+		"core/latest-posts",
+		"core/verse",
+		"core/cover",
+		"core/social-link",
+		"core/social-links",
+		"jetpack/contact-info",
+		"jetpack/email",
+		"jetpack/phone",
+		"jetpack/address"
```

Comment:
As we explore a more idea path to generate this list (or multiple lists that get combined) these probably shouldn't live in Gutenberg but instead, be a part of Gutenberg-Mobile. I left them here as an MVP approach.